### PR TITLE
Migrate 35up package

### PR DESCRIPTION
**Pull request by sourcecube**
This PR migrates the package https://github.com/fond-of/spryker-thirty-five-up 

Includes
- Renamed namespace
- Skipped propel tests
- Updated requirements
- Updated transfer object to include all properties

### CODE REVIEW

File: bundles/thirty-five-up/composer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "authors": [
+    {
+      "name": "Julian Zimmermann",
+      "email": "info@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.3",
+    "spryker/locale": "^2.2.0 || ^3.0.0",
+    "spryker/store": "^1.1.0"
```
Comment (sourcecube) [inline bundles/thirty-five-up/composer.json:14]: Added requirements

File: bundles/thirty-five-up/src/FondOfOryx/Shared/ThirtyFiveUp/Transfer/thirty_five_up.transfer.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <property name="thirtyFiveUpOrder" type="ThirtyFiveUpOrder"/>
+        <property name="items" singular="item" type="Item[]" />
+    </transfer>
+
+    <transfer name="Item">
+        <property name="abstractAttributes" type="array" singular="abstractAttribute" />
+        <property name="quantity" type="int" />
+        <property name="sku" type="string" />
+        <property name="fkSalesOrderItem" type="int" />
+    </transfer>
```
Comment (sourcecube) [inline bundles/thirty-five-up/src/FondOfOryx/Shared/ThirtyFiveUp/Transfer/thirty_five_up.transfer.xml:50]: Added

File: bundles/thirty-five-up/tests/FondOfOryx/Zed/ThirtyFiveUp/Communication/Plugin/Sales/AddThirtyFiveUpDataToSalesOrderItemExpanderPreSavePluginTest.php [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+        $this->thirtyFiveUpVendorMock = $this->getMockBuilder(ThirtyFiveUpVendorTransfer::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->plugin = new AddThirtyFiveUpDataToSalesOrderItemExpanderPreSavePlugin();
+    }
+
+    /**
+     * @skip Needs propel
```
Comment (sourcecube) [inline bundles/thirty-five-up/tests/FondOfOryx/Zed/ThirtyFiveUp/Communication/Plugin/Sales/AddThirtyFiveUpDataToSalesOrderItemExpanderPreSavePluginTest.php:88]: Added

File: bundles/thirty-five-up/tests/FondOfOryx/Zed/ThirtyFiveUp/Communication/Plugin/Sales/ThirtyFiveUpOrderExpanderPluginTest.php [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+             */
+            protected function getFacade(): AbstractFacade
+            {
+                return $this->facade;
+            }
+        };
+    }
+
+    /**
+     * @skip Needs propel
```
Comment (sourcecube) [inline bundles/thirty-five-up/tests/FondOfOryx/Zed/ThirtyFiveUp/Communication/Plugin/Sales/ThirtyFiveUpOrderExpanderPluginTest.php:92]: Added

File: bundles/thirty-five-up/tests/FondOfOryx/Zed/ThirtyFiveUp/Communication/Plugin/Sales/ThirtyFiveUpOrderExpanderPluginTest.php [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    {
+        $this->facadeMock->expects($this->once())->method('createThirtyFiveUpOrderFromQuote')->willReturn($this->quoteTransferMock);
+        $this->quoteTransferMock->expects($this->once())->method('getThirtyFiveUpOrder')->willReturn($this->thirtyFiveUpOrderTransferMock);
+        $this->thirtyFiveUpOrderTransferMock->expects($this->once())->method('getId')->willReturn(1);
+        $this->spySalesOrderEntityTransferMock->expects($this->once())->method('setFkThirtyFiveUpOrder');
+        $this->plugin->expand($this->spySalesOrderEntityTransferMock, $this->quoteTransferMock);
+    }
+
+    /**
+     * @skip Needs propel
```
Comment (sourcecube) [inline bundles/thirty-five-up/tests/FondOfOryx/Zed/ThirtyFiveUp/Communication/Plugin/Sales/ThirtyFiveUpOrderExpanderPluginTest.php:105]: Added

**Review by julianzimmermann — APPROVED** [review]

**Pull request merged**